docker-library/apache-php-mysql/sshd_config

```diff
@@ -0,0 +1,22 @@
+# This is ssh server systemwide configuration file.
```

Comment:
Keep all the values on the right side vertically align. You can use tab to padding the space.